adapters/medianet/medianet.go

```diff
@@ -106,6 +132,31 @@ func getMediaTypeForImp(impID string, imps []openrtb2.Imp) (openrtb_ext.BidType,
 	}
 }
 
+func extractFledge(a *adapter, bidResp openrtb2.BidResponse) []*openrtb_ext.FledgeAuctionConfig {
+	var fledgeAuctionConfigs []*openrtb_ext.FledgeAuctionConfig
+
+	var bidRespExt medianetRespExt
+	if err := json.Unmarshal(bidResp.Ext, &bidRespExt); err != nil {
+		return nil
+	}
```

Comment:
any particular reason to not return error from line 140

Reply:
Done, took error into consideration for assigning fledge config.